From: Punit Agrawal <punit@debian.org>
Date: Fri, 22 Sep 2017 18:01:21 +0100
Subject: gozilla: More robust input error handling

The check for argc only tested if it is zero rather than any value
less than equal 0. This led to segmentation fault when fed with
invalid input.

Fix this by erroring out if argc <= 0.
---
 gozilla/gozilla.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gozilla/gozilla.c b/gozilla/gozilla.c
index 22d2a95..050d228 100644
--- a/gozilla/gozilla.c
+++ b/gozilla/gozilla.c
@@ -273,7 +273,7 @@ main(int argc, char **argv)
 	 * Replace alias name.
 	 */
 	if (definition == NULL) {
-		if (argc == 0)
+		if (argc <= 0)
 			usage();
 		strbuf_puts(arg, argv[0]);
 		/*
